bundles/org.eclipse.passage.lic.hc/src/org/eclipse/passage/lic/internal/hc/i18n/MineMessages.java

```diff
@@ -14,18 +14,15 @@
 
 import org.eclipse.osgi.util.NLS;
 
-public class HcMessages extends NLS {
-	private static final String BUNDLE_NAME = "org.eclipse.passage.lic.internal.hc.i18n.HcMessages"; //$NON-NLS-1$
+public class MineMessages extends NLS {
+	private static final String BUNDLE_NAME = "org.eclipse.passage.lic.internal.hc.i18n.MineMessages"; //$NON-NLS-1$
 
 	public static String DecryptedConditions_no_transport_for_content_type;
 	public static String DecryptedConditions_reading_error;
 
 	public static String HttpClient_final_error_message;

```

Comment:
this key has no message in the .properties

Reply:
Indeed, it has bee migrated to `AccessMessages.HttpClient_failure`. Removed.